spec/services/prison_visits/api_spec.rb

```diff
@@ -30,16 +30,30 @@
       contact_email_address: "ada@test.example.com",
       contact_phone_no: "01154960222",
       slot_options: [
-        "2016-04-18T13:30/14:30", # (mon)
-        "2016-04-19T13:30/14:30", # (tue)
-        "2016-04-20T14:45/15:45"  # (wed)
+        "2016-04-25T13:30/14:30", # (mon)
+        "2016-04-26T13:30/14:30", # (tue)
+        "2016-04-27T14:45/15:45"  # (wed)
       ]
     }
   }
 
-  # Specs use vcr cassettes, no real calls are made
   before do
+    # Specs use vcr cassettes, no real calls are made
     WebMock.allow_net_connect!
+
+    # Needs to be set for all the requests, otherwise the recorded request's
+    # wont match
```

Comment:
I'd like a bit more explanation about the why of this, please--something about VCR being set to match on [:method, :uri, :host, :path, :body, :headers].